contents/tutorials/prevent-fouc-ab-tests.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Let's also add this as a question to the docs FAQ:  https://posthog.com/docs/experiments/common-questions

Reply:
Added with 0946f12ae